### ISSUE TITLE
Accessibility - ALT Text for Image on Step 4 Topic 6 Screen

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#42]
The image on the Step 4 Topic screen is missing ALT text.

**Comment by adriennerlong** [comment]
This image has been removed from the text. Please close on this issue.

**Comment by jfc3-dol** [comment]
@adriennerlong,

As a rule we will close issue once we see that it is fixed.

**Comment by jfc3-dol** [comment]
Closing this issues after verifying the Topic is no longer part to the Step.
